change/@fluentui-react-tabs-5328c68c-5b39-42af-a006-4fd2e3a2517a.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Modified role prop of CommandButton in Pivot.Base and Tabs.Base to incorporate link.role",
+  "packageName": "@fluentui/react-tabs",
+  "email": "hetanthakkar1@gmail.com",
+  "dependentChangeType": "none"
```

This should have type `prerelease`, or the change won't get published.
```suggestion
  "type": "prerelease",
  "comment": "Tabs: role prop of header button can now take tab.role value",
  "packageName": "@fluentui/react-tabs",
  "email": "hetanthakkar1@gmail.com",
  "dependentChangeType": "patch"
```